libs/blocks/global-navigation/base.css

```diff
@@ -14,7 +14,7 @@
   --feds-borderColor--light: #eaeaea;
   --feds-borderColor-menu--light: #e1e1e1;
   /* Top navigation - colors */
-  --feds-color-adobeBrand: #fa0f00;
+  --feds-color-adobeBrand: #EB1000;
```

Comment:
Should we also update https://github.com/adobecom/milo/blob/main/libs/blocks/global-navigation/global-navigation.js#L38?

Reply:
Yup checking currently, we should